lib/improver/utilities/spatial.py

```diff
@@ -85,7 +85,7 @@ def check_if_grid_is_equal_area(cube):
 
 
 def convert_distance_into_number_of_grid_cells(
-        cube, distance, max_distance_in_grid_cells):
+        cube, distance, max_distance_in_grid_cells=10**4):
```

Comment:
Within this function, we could consider whether the exception messages that are being generated might need some modification, i.e. the ones referring to "Neighbourhood processing", as we're now using this function outside of neighbourhood processing, and these exception messages could be confusing.